docs/api.asciidoc

```diff
@@ -102,7 +111,9 @@ Arguments:
   * `name`: A name for your span. **Required**.
   * `type`: The type of work eg. `db.postgresql.query`.
   * `context`: An instance of `Span::Context`.
-  * `&block`: An optional block to wrap with the span. The block is passed the span as an optional argument.
+  * `include_stacktrace`: Whether or not to collect a Stacktrace.
```

Comment:
Do you describe `Span::Context` somewhere?

There is a `ElasticAPM.build_context` description below, but you don't motivate what this context can be used for. Could you please clarify which context to use where and how to create?